### PR TITLE
Refactor/rework burger menu

### PR DESCRIPTION
**Pull request by nicole-eb**

### CODE REVIEW

File: apps/enmeshed/lib/account/app_drawer/drawer_main_page.dart [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+                  title: Text(context.l10n.drawer_helpAndFaq, style: Theme.of(context).textTheme.labelLarge),
+                ),
+              const Divider(indent: 16, endIndent: 16),
+              ListTile(
+                onTap: () => context
+                  ..pop()
+                  ..push('/legal-notice'),
+                shape: const RoundedRectangleBorder(borderRadius: borderRadius),
+                title: Text(context.l10n.legalNotice, style: Theme.of(context).textTheme.labelLarge),
+              ),
```
Comment (jkoenig134) [inline apps/enmeshed/lib/account/app_drawer/drawer_main_page.dart:68]: I see why this went wrong, but I just removed this in #260 .. please test out your PRs.

Comment (jkoenig134) [reply apps/enmeshed/lib/account/app_drawer/drawer_main_page.dart:68]: legal-notice it not available in this app.

Comment (nicole-eb) [reply apps/enmeshed/lib/account/app_drawer/drawer_main_page.dart:68]: oh ok!

Comment (jkoenig134) [reply apps/enmeshed/lib/account/app_drawer/drawer_main_page.dart:68]: No, stop! I fd up :D

Comment (jkoenig134) [reply apps/enmeshed/lib/account/app_drawer/drawer_main_page.dart:68]: I completely messed this up :D

**Review by jkoenig134 — APPROVED** [review]

File: apps/enmeshed/lib/account/app_drawer/drawer_main_page.dart (hunk 1 above)
Comment (nicole-eb) [reply apps/enmeshed/lib/account/app_drawer/drawer_main_page.dart:68]: whoopsie

**Pull request merged**